tests/e2e/standard/contacts/contact-summary.wdio-spec.js

```diff
@@ -249,132 +246,120 @@ describe('Contact summary info', () => {
     ]
   };
```

Comment:
What do you think about using factories to generate the contacts, places, and reports defined in the above lines of code?